modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc

```diff
@@ -19,22 +21,35 @@
 . Click **Delete**.
 
 . The project is now deleted from the **Projects** list and cluster.
+////
 
+To uninstall pipelines you need to first delete the pipeline configuration custom resource and then uninstall the pipeline operator. Deleting the Pipelines operator will not delete all the custom resources created by default when the operator is installed.
```

Comment:
the custom resource is called `config` (config.operator.tekton.dev). It is not short for pipeline configuration.

Reply:
> Deleting the Pipelines operator will not delete all the custom resources created by default when the operator is installed.

how about something like,
it should be `uninstalling the pipelines operator will not remove openshift-pipelines components ...`

"custom resource" might make an admin think about custom resources like tasks, pipelines etc.